fiadoc/utils.py

```diff
@@ -35,7 +35,10 @@ def duration_to_millisecond(s: str | None) -> dict[str, str | int] | None:
     {'_type': 'timedelta', 'milliseconds': 12345}
     """
     if s is None:
-        return None
+        return {
+            '_type': 'timedelta',
+            'milliseconds': 0
```

Comment:
This is questionable, returning 0 for unknown lap times could be confusing. None (null in json) might be preferable, but pydantic currently doesn't allow that. We may need to discuss with Jess, regarding the preferred output for Jolpica